demo/admin/package.json

```diff
@@ -33,7 +33,7 @@
         "@comet/cms-admin": "workspace:*",
         "@emotion/react": "^11.9.3",
         "@emotion/styled": "^11.9.3",
-        "@fontsource/roboto": "^4.5.5",
+        "@fontsource-variable/roboto-flex": "^5.0.14",
```

Comment:
This should be mentioned in the changeset, that the font also needs to be changed in the application.

Reply:
And/or mention the codemod added in https://github.com/vivid-planet/comet-upgrade/pull/11 (assuming it's not executed automatically)

Reply:
True, I forgot that. Done in 0f7136a561dc6d4699478a803c886e2fd68b7b2a